charts/signoz/templates/otel-collector-metrics/deployment.yaml

```
@@ -41,6 +41,8 @@ spec:
             - -c
             - until wget --user "${CLICKHOUSE_USER}:${CLICKHOUSE_PASSWORD}" --spider -q {{ include "clickhouse.httpUrl" $ }}{{ .endpoint }}; do echo -e "{{ .waitMessage }}"; sleep {{ .delay }}; done; echo -e "{{ .doneMessage }}";
           {{- end -}}
```

```suggestion
          {{- end }}
```

This is causing the next `resources` section to be concatted at the end of previous `command` array.